client/src/transmitter/index.ts

```diff
@@ -190,20 +193,22 @@ function start_recording(stream: MediaStream) {
   const connectTime = Date.now()
   connect(
     (openEvent) => {
-      setup_media_recorder_listener()
+      start_media_recorder()
```

Comment:
what if user clicks `stop_recording` before the socket connected?

Reply:
Nice.